### ISSUE TITLE
Linux 32 bits

### ISSUE THREAD

**Issue by Immortalin** [superpowers/superpowers#36]
Support 32-bits linux?

**Comment by elisee** [comment]
We've found that very few Linux users are still stuck on 32-bit so we thought maybe we can get away with one less build. Do you have an actual need for it?

**Comment by Immortalin** [comment]
@elisee yes...

**Comment by Immortalin** [comment]
@elisee does the additional build really make that much of a difference considering that you guys are already using a continous integration system?

**Comment by bilou84** [comment]
We plan to separate the launcher (the app to package superpowers) from everything else in the near future. At this point, the launcher should get very few updates so maybe we can manage one more build ?

One thing to note is that we have to manually build and upload every build.
Until recently, we also had to host every build (of every version) on our own server. So having one less build saved us both time and space on our humble server. Now that we use GitHub to host the builds, this problem is gone :)

**Comment by elisee** [comment]
@Immortalin ok, then let's have a Linux 32-bit build going forward, if it's truly useful for even one person then it's worth the trouble =). To explain a bit more, we used to have a fully automated build process for all 4 platforms but we have a couple issues right now:

 * Our build server is stuck on an older incompatible version of Node.js for a while longer because of legacy apps
 * electron-packager has a native Windows requirement for the Windows builds and installing Wine on our build server isn't possible either: https://github.com/maxogden/electron-packager#building-windows-apps-from-non-windows-platforms

So for the last couple of releases we've built them locally and uploaded them from our Internet connection with terrible upload rate. So earlier, the reason for doing as few builds as possible was disk space on our VPS. Now, it's upload speed :D.

As you mention, going forward, we might be able to use Travis to upload the builds to GitHub? cc @BaptisteDixneuf
